pyproject.toml

```diff
@@ -1,5 +1,153 @@
 [build-system]
-requires = ["setuptools", "wheel", "versioneer-518"]  # PEP 508 specifications.
+requires = ["hatchling>=1.8.0"]
+build-backend = "hatchling.build"
+
+[project]
+name = "ethyca-fides"
```

Comment:
```suggestion
name = "fidesctl"
```

Reply:
In this case the name is about to be changed so `ethyca-fides` is correct for where things are headed.